pkg/proxy/healthcheck/proxy_health.go

```diff
@@ -164,13 +200,13 @@ func (hs *ProxyHealthServer) Run(ctx context.Context) error {
 
 	listener, err := hs.listener.Listen(ctx, hs.addr)
 	if err != nil {
-		return fmt.Errorf("failed to start proxy healthz on %s: %v", hs.addr, err)
+		return fmt.Errorf("failed to start proxy healthz on %s: %w", hs.addr, err)
```

Comment:
I suspect the linter is forcing you to do this because you renamed the file, in which case you probably ought to do it as part of the rename commit. (Likewise the `_ *http.Request` below.)